### PR TITLE
cgitb stubs

### PR DESCRIPTION
**Pull request by CraftSpider**
Stubs for the cgitb module, 2 and 3

### CODE REVIEW

File: stdlib/2and3/cgitb.pyi [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+else:
+    _Path = Union[_T]
+
+
+def reset() -> str: ...
+def small(text: str) -> str: ...
+def strong(text: str) -> str: ...
+def grey(text: str) -> str: ...
+def lookup(name: str, frame: FrameType, locals: Dict[str, Any]) -> Tuple[Optional[str], Any]: ...
+def scanvars(reader: Callable[[], bytes], frame: FrameType, locals: Dict[str, Any]) -> List[Tuple[str, Optional[str], Any]]: ...
```
Comment (srittau) [inline stdlib/2and3/cgitb.pyi]: Please add `# undocumented` to all of the above.

File: stdlib/2and3/cgitb.pyi [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+def html(einfo: _ExcInfo, context: int = ...) -> str: ...
+def text(einfo: _ExcInfo, context: int = ...) -> str: ...
+
+class Hook:
```
Comment (srittau) [inline stdlib/2and3/cgitb.pyi]: Also `# undocumented`.

File: stdlib/2and3/cgitb.pyi [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    def __init__(self, display: int = ..., logdir: Optional[_Path[AnyStr]] = ..., context: int = ..., file: Optional[IO[str]] = ..., format: str = ...) -> None: ...
+    def __call__(self, etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
+    def handle(self, info: Optional[_ExcInfo] = ...) -> None: ...
+
+handler: Callable[[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]], None]
```
Comment (srittau) [inline stdlib/2and3/cgitb.pyi]: From the documentation it does not sound like this is meant to be overridden. So I think it makes more sense to use a straight `def` here in the stubs.

File: stdlib/2and3/cgitb.pyi [hunk 4]
```diff
@@ -13,21 +13,21 @@ else:
...
 
-class Hook:
+class Hook:  # undocumented
 
     def __init__(self, display: int = ..., logdir: Optional[_Path[AnyStr]] = ..., context: int = ..., file: Optional[IO[str]] = ..., format: str = ...) -> None: ...
     def __call__(self, etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
     def handle(self, info: Optional[_ExcInfo] = ...) -> None: ...
 
-handler: Callable[[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]], None]
+def handler(etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
```
Comment (srittau) [inline stdlib/2and3/cgitb.pyi]: This should just have one `info` argument, though (like `Hook.handle()`).

Comment (CraftSpider) [reply stdlib/2and3/cgitb.pyi]: Oh whoops, copied from the wrong source function (And I had just looked at the documentation, too)

File: stdlib/2and3/cgitb.pyi [hunk 5]
```diff
@@ -28,6 +28,6 @@ class Hook:  # undocumented
     def __call__(self, etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
     def handle(self, info: Optional[_ExcInfo] = ...) -> None: ...
 
-def handler(etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
+def handler(info: Optional[_ExcInfo]) -> None: ...
```
Comment (srittau) [inline stdlib/2and3/cgitb.pyi]: Just add `= ...`, and I am happy :)

Comment (CraftSpider) [reply stdlib/2and3/cgitb.pyi]: Done, sorry about that. I should be more careful editing in my early morning

**Pull request merged**